src/mame/mit/cadr_tv_control.cpp

```diff
@@ -0,0 +1,161 @@
+// license:BSD-3-Clause
+// copyright-holders:Wilbert Pol
+/**********************************************************************************
+
+CADR TV Control emulation
+
+The TV control contains sync ram which looks like it can be configured to
+generate output to different types of TV displays.
+
+This currently emulates a 768x939 monochrome display.
+
+Other information about displays:
+
+Color screen: 454x576, 16 colors, RGB.
+Grayscale monitor: 454x576, 16 scales of gray.
+'CPT' monitor: 768x896, monochrome?
+
+60MHz and 64Mhz clocks are mentioned in the docs, as well as 64.69Hz default refresh rate.
+
+Implementation based on description of the operation, not schematics.
+
+
+**********************************************************************************/
+#include "emu.h"
+#include "cadr_tv_control.h"
+
+#include "screen.h"
```

Comment:
You already included `screen.h` in the corresponding header – there’s no need to have it here as well.